app/cancel/get.njk

```diff
@@ -1,13 +1,13 @@
 {% extends "common/templates/layout.njk" %}
 
 {% block page_title %}
-  Your payment has been cancelled - GOV.UK Pay
+  Your {{ "mandate" if mandateType == 'ON_DEMAND' else "payment" }} has been cancelled - GOV.UK Pay
```

Comment:
I think we’d future-proof ourselves if this was `
"payment" if mandateType == 'ONE_OFF' else "mandate"` — it being a payment rather than a mandate is the exceptional case (and will ensure this code keeps working when we have `RECURRING` mandates or whatever)

Reply:
As discussed:
In case of `ONE_OFF` we want to display `payment` even when we cancel the one off mandate, because users don't have idea of mandate in one off payments.
That is why the condition `"payment" if mandateType == 'ONE_OFF' else "mandate"` should include all scenarios